src/StackExchange.Redis/ServerEndPoint.cs

```diff
@@ -458,7 +458,7 @@ internal async Task AutoConfigureAsync(PhysicalConnection? connection, ILogger?
             // But if GETs are disabled on this, do not fail the connection - we just don't get tiebreaker benefits
             if (Multiplexer.RawConfig.TryGetTieBreaker(out var tieBreakerKey) && Multiplexer.CommandMap.IsAvailable(RedisCommand.GET))
             {
-                log?.LogInformation($"{Format.ToString(EndPoint)}: Requesting tie-break (Key=\"{tieBreakerKey}\")...");
+                log?.LogInformationRequestingTieBreak(new LoggerExtensions.EndPointLogValue(EndPoint), tieBreakerKey.ToString());
```

Comment:
can this not be target-typed, i.e. `new(EndPoint)` ? also: might need `tieBreakerKey?.ToString()` and `string?` in the receiver

Reply:
we enter this code block only if `Multiplexer.RawConfig.TryGetTieBreaker(out var tieBreakerKey)` is true, I think the `tieBreakerKey` is not null here